packages/app/src/app/pages/common/Modals/StarterModal/index.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
```

Comment:
We should probably rename this to `TemplateModal` to make it less confusing for new people.

Reply:
Done :)